content/en/watchdog/insights.md

```diff
@@ -249,6 +250,23 @@ In the full side panel, you can see a timeseries graph about the performance met
 [1]: /real_user_monitoring/explorer/search/#facets
 [2]: /real_user_monitoring/browser/monitoring_page_performance/#event-timings-and-core-web-vitals
 [3]: /real_user_monitoring/browser/monitoring_page_performance/#monitoring-single-page-applications-spa
+{{% /tab %}}
+{{% tab "Synthetic Monitoring" %}}
+
+### Error outliers
+
+Error outliers in Synthetic Monitoring display unexpected behaviors and performance deviations. These anomalies provide insights into the reliability issues in your [Synthetic Browser Tests][101]. Identifying these error outliers helps you troubleshoot errors in failed test runs, enhancing debugging and reducing Mean Time To Resolution (MTTR).
```

Comment:
I noticed the linked page doesn't capitalize "browser tests," so I think it's probably easiest to stay consistent 🙂 
```suggestion
Error outliers in Synthetic Monitoring display unexpected behaviors and performance deviations. These anomalies provide insights into the reliability issues in your [Synthetic browser tests][101]. Identifying these error outliers helps you troubleshoot errors in failed test runs, enhancing debugging and reducing Mean Time To Resolution (MTTR).
```

Reply:
oh that one is suppose to be capitalized as it's a product :)